pybikes/bilbonbizi.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017, Alberto Varela <alberto@berriart.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+from lxml import etree
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes.utils import PyBikesScraper
+
+__all__ = ['BilbonBizi']
+
+class BilbonBizi(BikeShareSystem):
+    sync = True
+
+    meta = {
+        'system': 'BilbonBizi',
+        'company': ['Ayuntamiento de Bilbao']
+    }
```

Comment:
Since this API has a license (not that usual) can you add a `license` field similar to https://github.com/eskerda/pybikes/blob/master/pybikes/cyclocity.py#L35?